packages/flutter/test/material/page_test.dart

```diff
@@ -540,4 +540,79 @@ void main() {
     expect(find.text('Page 1'), isOnstage);
     expect(find.text('Page 2'), findsNothing);
   });
+
+  testWidgets('Back swipe dismiss interrupted by route push', (WidgetTester tester) async {
```

Comment:
Please add this comment
`// Regression test for https://github.com/flutter/flutter/issues/28728`